website/docs/docusaurus-core.md

```diff
@@ -228,6 +228,29 @@ const MyComponent = (props) => {
 };
 ```
 
+:::info Why do we use a "render function"?
+
+It's important to realize that the children of `<BrowserOnly>` is not a JSX element, but a function that _returns_ an element. This is a design decision. Consider this code:
```

Comment:
👍 

Maybe worth explaining too that it's a function that is executed as is and not a React component declared inline (bad practice)? (ie you can't call React hooks in this function)

It's up to you.

Reply:
I'll still keep it brief😄 Calling hooks will probably throw "invalid hook call" anyways, and once I find a better place to document this, we'll elaborate on that.